### PR TITLE
Small changes to JSON State Test format

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: fuzzing/statetest_maker.go [hunk 1]
```diff
@@ -146,9 +146,9 @@ func NewGstMaker() *GstMaker {
 			// The ENV portion
 			Number:     1,
 			GasLimit:   0x26e1f476fe1e22,
-			Difficulty: big.NewInt(0x20000),
+			Difficulty: big.NewInt(0x200000),
```
Comment (holiman) [inline fuzzing/statetest_maker.go:149]: Why? `0x20000` is the minimum allowed difficulty

File: fuzzing/statetest_maker.go [hunk 2]
```diff
@@ -146,9 +146,9 @@ func NewGstMaker() *GstMaker {
...
 			Coinbase:   common.HexToAddress("b94f5374fce5edbc8e2a8697c15331677e6ebf0b"),
-			Timestamp:  0x03e8,
+			Timestamp:  0x13e8,
```
Comment (holiman) [inline fuzzing/statetest_maker.go:151]: why?

File: fuzzing/statetest_maker.go (hunk 1 above)
Comment (chfast) [reply fuzzing/statetest_maker.go:149]: It's encoding issue. Maybe I can fix it on my side, but the official state tests add leading zeros in the encoding so the total number of chars is even.

**Comment by holiman** [comment]
Closing this, sender has been added and I think you fixed the hex-parsing on your side

**Pull request closed without merging**